packages/apps/shopify-app-react-router/src/server/types-contexts.ts

```diff
@@ -6,91 +6,48 @@ import type {
 import type {WebhookContext as IWebhookContext} from './authenticate/webhooks/types';
-import type {FlowContext as IFlowContext} from './authenticate/flow/types';
-import type {FulfillmentServiceContext as IFulfillmentServiceContext} from './authenticate/fulfillment-service/types';
 import type {
   AppProxyContext as IAppProxyContext,
   AppProxyContextWithSession as IAppProxyContextWithSession,
 } from './authenticate/public/appProxy/types';
 import type {CheckoutContext as ICheckoutContext} from './authenticate/public/checkout/types';
 import type {CustomerAccountContext as ICustomerAccountContext} from './authenticate/public/customer-account/types';
 import type {
-  AdminApiContextWithRest,
   AdminApiContext as IAdminApiContext,
   StorefrontContext as IStorefrontContext,
 } from './clients';
 
+// Direct re-exports for types without transformations
+export type {UnauthenticatedAdminContext} from './unauthenticated/admin/types';
+export type {UnauthenticatedStorefrontContext} from './unauthenticated/storefront/types';
+export type {FlowContext} from './authenticate/flow/types';
+export type {FulfillmentServiceContext} from './authenticate/fulfillment-service/types';
+export type {ScopesDetail} from './authenticate/admin/scope/types';
+export type {AdminApiContext} from './clients';
+
 type ShopifyConfig<App> =
   App extends ShopifyApp<infer Config extends AppConfigArg> ? Config : never;
 
-type ConfigComponents<Config> =
-  Config extends AppConfigArg<infer Resources, infer Storage, infer Future>
-    ? {resources: Resources; storage: Storage; future: Future}
-    : never;
-
 type DefaultApp = ShopifyApp<AppConfigArg>;
 
-export type UnauthenticatedAdminContext<App = DefaultApp> =
-  IUnauthenticatedAdminContext<
-    ShopifyConfig<App>,
-    ConfigComponents<ShopifyConfig<App>>['resources']
-  >;
-
-export type AdminContext<App = DefaultApp> = IAdminContext<
-  ShopifyConfig<App>,
-  ConfigComponents<ShopifyConfig<App>>['resources']
->;
-
-export type ScopesContext<_App = DefaultApp> = IScopesContext;
-export type ScopesDetail = IScopesDetail;
-
-export type UnauthenticatedStorefrontContext<_App = DefaultApp> =
-  IUnauthenticatedStorefrontContext;
-
-export type FlowContext<App = DefaultApp> = IFlowContext<
-  ShopifyConfig<App>,
-  ConfigComponents<ShopifyConfig<App>>['resources']
->;
-
-export type FulfillmentServiceContext<App = DefaultApp> =
-  IFulfillmentServiceContext<
-    ShopifyConfig<App>,
-    ConfigComponents<ShopifyConfig<App>>['resources']
-  >;
-
-export type AppProxyContext<App = DefaultApp> =
-  | IAppProxyContext
-  | IAppProxyContextWithSession<ShopifyConfig<App>>;
+// Types that need transformations or generic parameters
+export type AdminContext<App = DefaultApp> = IAdminContext<ShopifyConfig<App>>;
 
-export type CheckoutContext<_App = DefaultApp> = ICheckoutContext;
+export type ScopesContext = IScopesContext;
```

Comment:
This is grouped under "Types that need transformations or generic parameters", but this one doesn't need transformation or a generic Param? Worth double checking this file again?